source/reference/modules/studio/security.rst

```diff
@@ -469,13 +469,23 @@ Authorization
 ^^^^^^^^^^^^^
 Role Mappings
 ^^^^^^^^^^^^^
-Users only see the items that they have been granted access to based on the permissions granted to the Role they have been assigned to. The role mappings configuration file defines the mapping between the group that the user belongs to and the studio authoring role. To modify the role mappings, click on |projectTools| from the bottom of the *Sidebar*, then click on **Configuration** and select **Role Mappings** from the list.
+Users are allowed to perform actions on the items that they have been granted access to based on the permissions granted
+to the role they have been assigned to. Note that all users are given read permission to the entire project/site
+regardless of the role that they are assigned.
+
+The role mappings configuration file defines a list of roles available in the project/site and the mapping between the
```

Comment:
The role mappings configuration file maps user groups to one or more roles which then get a set of permissions within a project.